app/default/routes.py

```diff
@@ -65,11 +68,12 @@ def new(account_id):
         url=f"{Config.APPLICATION_STORE_API_HOST}/applications",
         json={
             "account_id": account_id,
-            "round_id": request.form["round_id"] or Config.DEFAULT_ROUND_ID,
-            "fund_id": request.form["fund_id"] or Config.DEFAULT_FUND_ID,
+            "round_id": Config.DEFAULT_ROUND_ID,
+            "fund_id": Config.DEFAULT_FUND_ID,
```

Comment:
Should we keep the previous fallback, which was designed to allow multiple funds passed by query string?